### DISCUSSION TITLE
User verification emails from SendGrid

### DISCUSSION THREAD

**Discussion by richardltc** [pocketbase/pocketbase#228] [Q&A] [answered]
Hi,

After receiving a user verification email from SendGrid, I noticed that the URL, when clicking on the button, would go to some kind of sendgrid link, as apposed to the default localhost settings that I have in PocketBase:

https://u28........ct.sendgrid.net/ls/click?upn=cRmB6570Zm6f3tEwZO...................

I'm presuming this is some kind of SendGrid issue?

Richard

**Comment by ganigeorgiev** [comment, marked as answer]
This is common for mail providers. It still should work though - usually these links are redirecting to the actual ones.

They usually replace the links inside the emails with custom redirect ones so that they can track for example who clicked on the link (I'm not sure if SendGrid has option to disable that).

**Reply by ganigeorgiev** [reply]
There should be a "Click tracking" toggle somewhere in the SendGrid settings - https://docs.sendgrid.com/ui/account-and-settings/tracking#click-tracking

**Reply by richardltc** [reply]
That solved it, although the button, in the email, didn't actually open another tab, so I had to copy and paste the link?

Richard

**Reply by ganigeorgiev** [reply]
I've just added `target="_blank"` to all email links so that they can open in a new tab.
Currently it is only in master but It will be included in the next minor release somewhere around next week.

**Comment by richardltc** [comment]
Sounds perfect :) Thank you
